packages/crayons-core/src/components/timepicker/timepicker.scss

```diff
@@ -1,7 +1,3 @@
-/**
-  @prop --fw-hint-color: Color of the hint text.
-  @prop --fw-warning-color: Color of the warning text.
-  @prop --fw-error-color: Color of the error text.
-*/
-
-@import '../../function-components/field-control.scss';
+.timepicker {
```

Comment:
why do we need a `block` display here ?. It would mean, we cannot have other components inline with this.

Reply:
Without the `block` element the consuming component can't specify the width or the `margin/padding` for the component.

Reply:
I can be either a `block` or `inline-block` since all the other components are block elements i have used the same here.